eva-accession-release/src/main/java/uk/ac/ebi/eva/accession/release/batch/io/VariantMongoAggregationReader.java

```diff
@@ -91,7 +90,7 @@
 
     protected static final String SUPPORTED_BY_EVIDENCE_FIELD = "evidence";
 
-    protected static final String MAPPING_WEIGHT_FIELD = "mapWeight";
+    public static final String MAPPING_WEIGHT_FIELD = "mapWeight";
```

Comment:
because I moved the io classes to subfolders and they use this key (AccessionedVariantMongoReader checks mapWeight is null, multimap readers check it's not).